doc/docs/modules/ROOT/pages/producer-configuration.adoc

```diff
@@ -45,6 +46,12 @@ You can tune the custom property `kafka.topic.discovery.polling.interval` in ord
 periodically check for new topics into the Kafka cluster so the plugin will be able
 to send events to the defined topics.
 
+
+With `kafka.streams.log.compaction.strategy=delete` will be generated a sequence of unique keys with Neo4j Streams Source.
+instead with `kafka.streams.log.compaction.strategy=compact` the keys will be adapted to enable
+https://kafka.apache.org/documentation.html#compaction[Log Compaction] on the Kafka side.
+Namely, the operations which will involve the same nodes or relationships, will have the same key.
```

Comment:
State what the default is if you don't choose, which will be most users.    Line above suggests that it's delete?

Also, why do we call it delete?  This sounds like something's getting deleted....now when logs get compacted to 500mb (or whatever) that's probably true, things get deleted.  But maybe this should be called "unique" or something?  (Because the IDs end up unique and not compact-able) I'm not sure.  Which is the least confusing for users?

Reply:
I chose these two values to match the topic config `cleanup.policy` which has compact / delete values to enable log compaction or not. Of course, the term unique sounds good.
So, If you want, I can change `delete` to `unique`

Reply:
I'm ok with the decision as it is, just make it clear in the docs.

Reply:
Ok, i've added a note in the `.adoc`